pkg/kdump/config-client.js

```diff
@@ -203,6 +203,8 @@ export class ConfigFile {
             this.settings._internal.core_collector.value &&
             (this.settings._internal.core_collector.value.split(" ").indexOf("-c") != -1)
         );
+
+        this.settings.core_collector = this.settings._internal?.core_collector.value ?? defaultCoreCollector;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19430-20231114-105301-678eacaf-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)